public/docs/_examples/style-guide/ts/03-03/app/shared/hero.model.ts

```diff
@@ -1,7 +1,9 @@
-// #docregion
-// #docregion example
 export class Hero {
   name: string;
   power: string;
 }
-// #enddocregion example
+
```

Comment:
If we are going to have an interface for the class, I suggest it should be listed above the class and the class should implement it